### PR TITLE
Make install script ssh as root (not current host user)

### PR DESCRIPTION
**Pull request by gjvanouwendorp**
#3

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-ssh rem "cp /usr/share/remarkable/templates/templates.json /usr/share/remarkable/templates/templates.backup.json"
+ssh root@rem "cp /usr/share/remarkable/templates/templates.json /usr/share/remarkable/templates/templates.backup.json"
+
 # Replacing the templates.json on the device with merged one
-scp templates.merged.json rem:/usr/share/remarkable/templates/templates.json
+scp templates.merged.json root@rem:/usr/share/remarkable/templates/templates.json
+
 # Restarting the GUI service
-ssh rem "systemctl restart xochitl"
+ssh root@rem "systemctl restart xochitl"
```
Comment (Eeems) [inline install.sh:17]:
```suggestion
ssh root@10.11.99.1 "systemctl restart xochitl"
```

File: install.sh [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 # Adding the custom templates to templates.json
 jq -n '{ templates: [ inputs.templates ] | add | unique_by(.name)}' templates.json templates.addition.json > templates.merged.json
+
 # Taking a backup of the templates.json in any case
-ssh rem "cp /usr/share/remarkable/templates/templates.json /usr/share/remarkable/templates/templates.backup.json"
+ssh root@rem "cp /usr/share/remarkable/templates/templates.json /usr/share/remarkable/templates/templates.backup.json"
+
 # Replacing the templates.json on the device with merged one
-scp templates.merged.json rem:/usr/share/remarkable/templates/templates.json
+scp templates.merged.json root@rem:/usr/share/remarkable/templates/templates.json
```
Comment (Eeems) [inline install.sh:14]:
```suggestion
scp templates.merged.json root@10.11.99.1:/usr/share/remarkable/templates/templates.json
```

File: install.sh [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
 # Receiving the current templates.json to add our custom templates
-scp rem:/usr/share/remarkable/templates/templates.json ./templates.json
+scp root@rem:/usr/share/remarkable/templates/templates.json ./templates.json
+
 # Adding the custom templates to templates.json
 jq -n '{ templates: [ inputs.templates ] | add | unique_by(.name)}' templates.json templates.addition.json > templates.merged.json
+
 # Taking a backup of the templates.json in any case
-ssh rem "cp /usr/share/remarkable/templates/templates.json /usr/share/remarkable/templates/templates.backup.json"
+ssh root@rem "cp /usr/share/remarkable/templates/templates.json /usr/share/remarkable/templates/templates.backup.json"
```
Comment (Eeems) [inline install.sh:11]:
```suggestion
ssh root@10.11.99.1 "cp /usr/share/remarkable/templates/templates.json /usr/share/remarkable/templates/templates.backup.json"
```

File: install.sh [hunk 4]
```diff
@@ -1,13 +1,18 @@
 # Sending the PNG files to reMarkable
-scp templates/pngs/*.png rem:/usr/share/remarkable/templates
+scp templates/pngs/*.png root@rem:/usr/share/remarkable/templates
+
 # Receiving the current templates.json to add our custom templates
-scp rem:/usr/share/remarkable/templates/templates.json ./templates.json
+scp root@rem:/usr/share/remarkable/templates/templates.json ./templates.json
```
Comment (Eeems) [inline install.sh:5]:
```suggestion
scp root@10.11.99.1:/usr/share/remarkable/templates/templates.json ./templates.json
```

File: install.sh [hunk 5]
```diff
@@ -1,13 +1,18 @@
 # Sending the PNG files to reMarkable
-scp templates/pngs/*.png rem:/usr/share/remarkable/templates
+scp templates/pngs/*.png root@rem:/usr/share/remarkable/templates
```
Comment (Eeems) [inline install.sh:2]:
```suggestion
scp templates/pngs/*.png root@10.11.99.1:/usr/share/remarkable/templates
```

**Review by Eeems — COMMENTED** [review]
It might be better to just use the USB IP address instead of assuming that the user has setup `rem` in their config file. Especially since the guide linked in the readme doesn't have any information about doing so.

**Comment by msencer** [comment]
Please see #9 to resolve this, wdyt?

**Comment by Eeems** [comment]
> Please see #9 to resolve this, wdyt?

That looks like it would be good. You'd want to update the readme as well ;)

**Pull request closed without merging**